cmd/openshift-install/destroy.go

```diff
@@ -112,6 +112,11 @@ func runDestroyCmd(directory string, reportQuota bool) error {
 	// ensure capi etcd data store is cleaned up
 	clusterapi.System().CleanEtcd()
```

Comment:
We don't need this part anymore, if we're removing the whole dir right after.

Reply:
Good point. We can do that as a follow-up. I also want to replace the deprecated `github.com/pkg/errors` dep.

Reply:
Follow-up here https://github.com/openshift/installer/pull/8720